ArduCopter/system.cpp

```diff
@@ -148,8 +148,26 @@ void Copter::init_ardupilot()
     }
 
 #if FRSKY_TELEM_ENABLED == ENABLED
-    // setup frsky
-    frsky_telemetry.init(serial_manager);
+    // setup frsky, and pass a number of parameters to the library
+    frsky_telemetry.init(serial_manager, FIRMWARE_STRING, FRAME_CONFIG_STRING, 
+#if (FRAME_CONFIG == QUAD_FRAME)
```

Comment:
Since there's similar code in GCS_Mavlink.cpp and to some extent in Copter.h, how would you suggest consolidating?